pipelines/migration/br_rj_riodejaneiro_gtfs/utils.py

```
@@ -375,12 +358,17 @@
         "tipo_os": "tipo_os",
     }
 
-    ordem_servico_trajeto_alternativo = ordem_servico_trajeto_alternativo.rename(
-        columns=alt_columns
-    )
+    for sheet_index, sheet_name in sheets:
```

Ruff (0.8.2)

Loop control variable `sheet_index` not used within loop body

Rename unused `sheet_index` to `_sheet_index`

(B007)